nixos/modules/hardware/decklink.nix

```
@@ -0,0 +1,17 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.hardware.decklink;
+  kernelPackages = config.boot.kernelPackages;
+in
+{
+  options.hardware.decklink.enable = mkEnableOption "hardware support for the Blackmagic Design Decklink audio/video interfaces";
+
+  config = mkIf cfg.enable {
+    boot.kernelModules = [ "blackmagic" "blackmagic-io" "snd_blackmagic-io" ];
+    boot.extraModulePackages = [ kernelPackages.decklink ];
+    systemd.packages = [ pkgs.blackmagicDesktopVideo ];
```

```suggestion
    systemd.packages = [ pkgs.blackmagicDesktopVideo ];
    systemd.services.DecklinkVideoHelper.wantedBy = [ "multi-user.target" ];
```

See https://github.com/NixOS/nixpkgs/issues/81138 for details.

---

It seem not to work yet.
```
[root@gaming:~]# lspci | grep Blackmagic
05:00.0 Multimedia video controller: Blackmagic Design Intensity Pro 4K
(this did work without the driver)
[root@gaming:~]# lsmod | grep blackmagic

[root@gaming:~]# systemctl status DecklinkVideoHelper
Unit DecklinkVideoHelper.service could not be found.
```